core/src/main/java/jenkins/views/FullHeader.java

```diff
@@ -13,4 +19,16 @@ public abstract class FullHeader extends Header {
     public boolean isCompatible() {
         return true;
     }
+
+    /**
+     * @return a list of {@link Action} to show in the header, defaults to {@link hudson.model.RootAction} extensions
+     */
+    @Restricted(NoExternalUse.class)
```

Comment:
It would be better if this is method is part of the Header class. Right now this means that only a full header can make use of it but actually it is better if I can make use of it in a PartialHeader as this would make the header incompatible in case of a future change.

Reply:
Done.